### PR TITLE
[DOCS] fixes issue number 91889 - missing [discrete] header

### PR DESCRIPTION
**Pull request by amyjtechwriter**
Fixes issue #91889

### CODE REVIEW

File: docs/plugins/plugin-script.asciidoc [hunk 1]
```diff
@@ -188,7 +188,7 @@ purge the configuration files while removing a plugin, use `-p` or `--purge`.
 This can option can be used after a plugin is removed to remove any lingering
 configuration files.
 
-[[removing-multiple-plugins]]
+[discrete]
```
Comment (abdonpijpelink) [inline docs/plugins/plugin-script.asciidoc:191]:
```suggestion
[discrete]
[[removing-multiple-plugins]]
```

Comment (abdonpijpelink) [reply docs/plugins/plugin-script.asciidoc:191]: I wouldn't remove the ID. It helps us link to this section directly from other places in the docs.

**Review by abdonpijpelink — CHANGES_REQUESTED** [review]

File: docs/plugins/plugin-script.asciidoc (hunk 1 above)
Comment (amyjtechwriter) [reply docs/plugins/plugin-script.asciidoc:191]: Ah ok, got it. Thanks for catching that. I'll commit your suggestion.

**Review by abdonpijpelink — APPROVED** [review]
Thanks @amyjtechwriter ! LGTM

**Pull request merged**